footer.php

```diff
@@ -282,6 +283,10 @@ function make_navbutton($imgsrc, $bgcolour, $title, $url, $newwindow = false) {
         }
         $output .= 'margin-right: 5px;" width="50" height="50" /></a>';
     } else {
+		$formClass = 'navbuttontext';
+		if ($classes != "") {
+		    $formClass .= " " . $classes;
+		}
```

Comment:
Please indent with spaces, not tabs.

Also, the comparison does not make sense - the default for the $classes parameter is null, but you then check it against "" to see if it has been specified. Far better to just write 'if ($classes) {' instead, as that is more consistent.